bin/hdfcoinc/pycbc_plot_psd_timefreq

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2015 Miriam Cabero Mueller
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Plot the variation of the amplitude spectral density (ASD) over time
+using the PSD hdf file generated by the pipeline
+"""
+
+import logging
+import argparse
+import numpy
+import h5py
+import sys
+import matplotlib
+matplotlib.use('agg')
+import pylab
+import pycbc.results
+from matplotlib.colors import LogNorm
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--psd-file', required=True,
+                    help='HDF file containing the PSDs.')
+parser.add_argument('--output-file', required=True,
+                    help='Name for the output plot.')
+parser.add_argument('--rel-median', choices=['True','False'], default=False, 
```

Comment:
You don't need to do the choices=[] thing here. This is a boolean option so you can use action='store_true'. Look for similar options for instance in pycbc_plot_singles_vs_params. Also, I think a better name for the option would be --normalize or --normalize-over-time.